packages/styling/src/styles/CommonStyles.ts

```diff
@@ -19,3 +23,14 @@ export const ScreenWidthMinUhfMobile = 768;
 export function getScreenSelector(min: number, max: number): string {
   return `@media only screen and (min-width: ${min}px) and (max-width: ${max}px)`;
 }
+
+/**
+ * The style which turns off high contrast adjustment in (only) Edge Chromium browser.
+ */
+export function getEdgeChromiumNoHighContrastAdjustSelector(): { [EdgeChromiumHighContrastSelector]: IStyle } {
```

Comment:
In the interest of bundle size, I wonder if we should use a shorter name for this?

Reply:
Also wonder if the return type should be IRawStyle.

Reply:
the function name doesn't affect bundle size as it can be minified. the size increase is mainly due to the styling code I added to checkbox, which is an unfortunate side-effect with making these fixes in our code :(

Reply:
Updated typing to `IRawStyle`

Reply:
Actually function names are not minifiable in many cases, including when exported. (Class method names are also not minifiable.)

Reply:
i verified in our `test-bundle` and it's being minified. can you elaborate when exactly you see it's not?

Reply:
Hmm I just looked into it and I guess I was incorrect in assuming that exported names would be handled the same as private members--searching for a similar example `getFocusOutlineStyle` in [this published version](https://unpkg.com/office-ui-fabric-react@7.110.5/dist/office-ui-fabric-react.min.js), all I find is `o.d(t,"getFocusOutlineStyle",(function(){return io}))`. So I guess it uses some optimization so exported things can still be referenced by name, but the name is minified. Sorry about the confusion.